packages/vite/src/node/ssr/ssrExternal.ts

```diff
@@ -120,13 +120,17 @@ export function createIsConfiguredAsSsrExternal(
   return (id: string) => {
     const { ssr } = config
     if (ssr) {
-      if (ssr.external?.includes(id)) {
+      const pkgName = getNpmPackageName(id)
```

Comment:
getNpmPackageName returns `''` for `id='/some/path/with/leading/slash'` which is falsy and returns undefined then. This is what we want, right?

Reply:
Only bare imports reach this check, so this is not possible 👍🏼